### PR TITLE
sip: add tests for sip over tcp

### PR DESCRIPTION
**Pull request by glongo**
Tests for sip tcp parser.

### CODE REVIEW

File: tests/sip-tcp-body-frames/README.md
Comment (inashivb) [inline tests/sip-tcp-body-frames/README.md:1]: Could you please add the source of PCAP file too? Script if you used any to create it yourself.
Thank you for your work! :)

Comment (glongo) [reply tests/sip-tcp-body-frames/README.md:1]: I've written some  C code that exchanges sip messages crafted manually.
Should I add them?

Comment (jasonish) [reply tests/sip-tcp-body-frames/README.md:1]: If they're small, or a link to a repo if they exist there any case we need to make some minor modifications. Thanks.

**Comment by glongo** [comment]
Please find code attached, pcap is generated by running both sip server and client, and capturing packets with tcpdump or wireshark.

[sip.zip](https://github.com/OISF/suricata-verify/files/11413222/sip.zip)

**Comment by inashivb** [comment]
> Please find code attached, pcap is generated by running both sip server and client, and capturing packets with tcpdump or wireshark.
> 
> [sip.zip](https://github.com/OISF/suricata-verify/files/11413222/sip.zip)

Thank you, @glongo ! Could you please commit the relevant files with the tests that they're intended for? It helps in the future if someone wants to replicate the same test as you or wants to understand how your pcap was created.
e.g. https://github.com/OISF/suricata-verify/tree/master/tests/ftp/ftp-too-long-command

**Pull request closed without merging**